CHANGELOG.md

```
@@ -9,6 +9,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- Add new changes here -->
 
+## [2.0.0] - Unreleased
+
+### Changed
+
+#### Encapsulation — Private RequestMeta Fields (#102)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)